CHANGELOG.md

```
@@ -1,5 +1,14 @@
 # Changelog
 
+## [1.0.19] - 2025-08-02
+
+**Chores**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)